### PR TITLE
ci(npm): add release for npm

### PR DESCRIPTION
**Pull request by elisechant**

### CODE REVIEW

**Comment by elisechant** [comment]
we need to out `/dist` to npm, but for now `/src` is ok for the stage that our project is in

File: package.json [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
-  "private": true,
+  "name": "@gov.au/datavizkit",
+  "version": "0.0.3-development",
+  "description": "A fully featured data visualisation toolset built on top of Highcharts and React.",
+  "keywords": [
+    "highcharts",
+    "dataviz"
+  ],
   "license": "MIT",
+  "author": "Digital Transformation Agency",
```
Comment (dominikwilkowski) [inline package.json:10]: we have the following bits in our `package.json` files:
```
"author": {
	"name": "Commonwealth of Australia, DTA",
	"email": "guides@digital.gov.au",
	"url": "https://www.dta.gov.au/"
},
"contributors": [
	{
		"name": "Dominik Wilkowski",
		"email": "hi@dominik-wilkowski.com",
		"url": "https://dominik-wilkowski.com/"
	},
	{
		"name": "Simon Pascal Klein",
		"email": "klepas@klepas.org",
		"url": "http://klepas.org/"
	}
],
```

File: package.json [hunk 2]
```diff
@@ -42,16 +57,17 @@
...
     "http-proxy-middleware": "0.17.3",
+    "husky": "^0.13.3",
     "jest": "18.1.0",
     "json-loader": "0.5.4",
     "object-assign": "4.1.1",
     "postcss-loader": "1.2.2",
     "promise": "7.1.1",
     "react-dev-utils": "^0.5.2",
     "react-styleguidist": "^5.0.2",
+    "semantic-release": "^6.3.2",
```
Comment (dominikwilkowski) [inline package.json:70]: you've got all dependencies fixed and those new ones are now ranges?

**Review by dominikwilkowski — COMMENTED** [review]
Just some dep questions... Looks good

**Pull request merged**